packages/documentation-component/package.json

```diff
@@ -29,28 +29,43 @@
   "scripts": {
     "start": "tsc --watch",
     "build": "tsc",
+    "buildRollup": "rimraf lib && rollup -c --compact",
     "lint-check": "tslint -c ../../tslint.json --project tsconfig.json --format verbose && prettier --config ../../.prettierrc --ignore-path ../../.prettierignore --check '**/*.{ts,tsx,js,jsx,json,html,css,yaml}'",
     "lint-fix": "tslint -c ../../tslint.json --project tsconfig.json --format verbose --fix; prettier --config ../../.prettierrc --ignore-path ../../.prettierignore --write '**/*.{ts,tsx,js,jsx,json,html,css,yaml}'",
     "prepare": "npm run build"
   },
   "dependencies": {
     "constate": "^1.1.1",
     "deepmerge": "^3.3.0",
-    "uuid": "3.3.2",
-    "react": "^16.8.6"
+    "react": "^16.8.6",
+    "uuid": "3.3.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.3.3",
     "@types/react": "16.8.20",
     "@types/uuid": "^3.4.5",
+    "@wessberg/rollup-plugin-ts": "^1.1.62",
     "prettier": "^1.18.2",
+    "rollup": "^1.17.0",
+    "rollup-plugin-cleaner": "^1.0.0",
```

Comment:
If rollup always will use those new packages, maybe better idea will be extract it to root package and only install in package needed for appropriate package?